.woodpecker.star

```diff
@@ -1474,26 +1482,33 @@ def logTracingResults():
 def dockerReleases(ctx):
     pipelines = []
     docker_repos = []
-    build_type = "daily"
-
-    # dockerhub repo
-    #  - "opencloudeu/opencloud-rolling"
-    repo = docker_repo_slug + "-rolling"
-    docker_repos.append(repo)
+    build_type = ""
 
-    # production release repo
     if ctx.build.event == "tag":
         tag = ctx.build.ref.replace("refs/tags/v", "").lower()
-        for prod_tag in PRODUCTION_RELEASE_TAGS:
+
+        is_production = False
+        for prod_tag in config["dockerReleases"]["production"]["tags"]:
             if tag.startswith(prod_tag):
-                docker_repos.append(docker_repo_slug)
+                is_production = True
                 break
 
+        # Tags starting with 2.0 indicate a production release → push to opencloudeu/opencloud
```

Comment:
```suggestion
```

IMO this is still misleading. Let's remove it completely. Whether this is a rolling or a production release is only defined through the tags listed in `config["dockerReleases"]["production"]["tags"]`